image/scripts/dev/installLatestK8sCesSetup.sh

```diff
@@ -16,6 +16,9 @@ helm_registry_username=${9}
 helm_registry_password=${10}
 helm_registry_url=${11}
 
+# set environment for helm and kubectl
+export KUBECONFIG=~/.kube/k3ces.local
```

Comment:
Should we add the "~/.kube/config" to the KUBECONFIG? So that it keeps to be the first place where kubectl looks for the config?